components/DrawControls.svelte

```diff
@@ -143,7 +143,7 @@
       currentMapHover.set(null);
     });
 
-    currentlyEditing.subscribe((id) => {
+    openFromSidebar.subscribe((id) => {
       if (id) {
         let feature = $gjScheme.features.find((f) => f.id == id);
         // Act like we've selected the object. (This is irrelevant for points.)
```

Comment:
Actually we have a pre-existing bug that's still here. If you edit a point from the sidebar, you can't drag the point as you should be able to. Not sure why yet